includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2097,6 +2097,17 @@ public function add_order_actions( $actions ) {
 			return $actions;
 		}
 
+		// if order is already completed, we shouldn't capture the charge anymore.
+		if (
```

Comment:
Can we put `if ( ... ) {` on one line, please?

Reply:
fixed in [222979d](https://github.com/Automattic/woocommerce-payments/pull/5285/commits/222979d301077252af3e6c02e12c16703d6851aa). thx